spec/game_spec.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require '../lib/game'
+
+RSpec.describe Game do
+  let(:game) { Game.new }
+
+  describe '#valid_move?' do
+    it 'returns true/false based on whether the position is already occupied' do
+      board = [' ', ' ', ' ', ' ', 'X', ' ', ' ', ' ', ' ']
+      index = 0
+      expect(game.valid_move?(board, index)).to be_truthy
+
+      index = 4
+      expect(game.valid_move?(board, index)).to be_falsey
+    end
+  end
+
+  describe '#position_taken?' do
+    it 'returns true/false based on whether the position is taken' do
+      board = [' ', ' ', 'O', ' ', 'X', ' ', ' ', ' ', ' ']
+      index = 2
+      expect(game.position_taken?(board, index)).to be_truthy
+
+      index = 5
+      expect(game.position_taken?(board, index)).to be_falsey
+    end
+  end
+
+  describe '#won?' do
+    it 'returns true/false if there winner or not' do
+      board_false = [' ', ' ', 'O', ' ', 'X', ' ', ' ', ' ', ' ']
+      board_true = [' ', ' ', '0', ' ', '0', ' ', '0', ' ', ' ']
+      expect(game.won?(board_false)).to be_falsey
+      expect(game.won?(board_true)).to be_truthy
+    end
+  end
+
+  describe '#full?' do
+    it 'return true/false if board is full or not' do
+      turn_count = 10
+      expect(game.full?(turn_count)).to be_truthy
+      turn_count = 5
```

Comment:
**[REQUIRED]**
This test is failing. We need to fix it.
![Screenshot from 2019-09-27 09-57-37](https://user-images.githubusercontent.com/30977845/65770886-6792a900-e10d-11e9-82b5-c25bb20c4594.png)

Reply:
The test is not failing in our tests!